src/reducers/snapshot.js

```diff
@@ -25,6 +25,10 @@ export const snapshotState = {
   dialogIsOpen: false,
   // for snapshot creation
   snapshotRequest: defaultSnapshotRequest,
+  // for snapshot export to workspace
+  exportIsProcessing: false,
```

Comment:
you could track exportIsRequested or submitted which would track when the user has clicked